_posts/2020-02-10-q1-2020-update.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Bisq Q1 2020 Update: Reorganization"
+author: Chris Beams
+excerpt: "After a year of milestone events, Bisq is implementing more purposeful management of priorities and funds to continue building its DAO to handle more contributors and more complexity. <br><br>"
+en-only: true
+---
+
+<div class='responsive-youtube-container'>
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/oImSdIAsywA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+
+The year 2019 was, by almost any reasonable measure, an incredibly successful year for the Bisq network. 
+
+Here is a sampling of the milestones achieved:
+
+- v1.0 launched after 4+ years of work
+- Bisq DAO launched as perhaps the first and only bitcoin-based DAO
+    - Of 20 100 total trades done since the DAO launch in 2019, 8 589 used BSQ to pay trading fees
```

Comment:
Any reason to do thousands separation as a space here and throughout the post instead of a comma? Europeans tend to use a period vs a comma, but this format seems jarring to all audiences. Perhaps I'm just less familiar with it. But would prefer comma for ease of parsing.

Reply:
It was an attempt to address that very difference in convention (comma v period for thousands separator)...I don't love it but figured avoiding periods and commas altogether might be more acceptable to both audiences.

I was going to argue keeping the spaces, but considering I couldn't [stick to my own convention](https://github.com/bisq-network/bisq-website/pull/342#discussion_r378441705), I've just switched all numbers in this post to use commas. I'll make another PR to switch past posts back to commas for consistency.